sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -770,48 +767,6 @@ private ConnectionOptions getConnectionOptions() {
         }
     }
 
-    private ProxyOptions getDefaultProxyConfiguration(Configuration configuration) {
-        ProxyAuthenticationType authentication = ProxyAuthenticationType.NONE;
-        if (proxyOptions != null) {
-            authentication = proxyOptions.getAuthentication();
```

Comment:
Information for other reviewers: here the authentication is never read correctly from the configuration. The correctly logic is moved to the `ProxyOptions.fromConfiguration()`.